pkg/bootstrapper/bootstrapper.go

```diff
@@ -116,11 +116,8 @@ type winNodeBootstrapper struct {
 	ignitionFilePath string
 	//initialKubeletPath is the path to the kubelet that we'll be using to bootstrap this node
 	initialKubeletPath string
-	// TODO: When more services are added consider decomposing the services to a separate Service struct with common functions
-	// kubeletSVC is a pointer to the kubelet Windows service object
-	kubeletSVC *mgr.Service
-	// svcMgr is used to interact with the Windows service API
-	svcMgr *mgr.Mgr
+	// service struct holds all service specific information
+	service *service
```

Comment:
It seems like this holds all the service specific information strictly for the kubelet service